services/src/main/java/uk/gov/moj/sdt/services/SubmitQueryService.java

```diff
@@ -414,18 +424,34 @@ private void sendRequestToTargetApp(final ISubmitQueryRequest submitQueryRequest
         long connectionTimeOut = 0;
 
         if (requestTimeOutParam != null) {
-            requestTimeOut = Long.valueOf(requestTimeOutParam.getValue());
+            requestTimeOut = Long.parseLong(requestTimeOutParam.getValue());
         }
 
         if (connectionTimeOutParam != null) {
-            connectionTimeOut = Long.valueOf(connectionTimeOutParam.getValue());
+            connectionTimeOut = Long.parseLong(connectionTimeOutParam.getValue());
         }
 
         LOGGER.debug("Send submit query request to target application");
 
-        this.requestConsumer.submitQuery(submitQueryRequest, connectionTimeOut, requestTimeOut);
+
+// TODO: uncomment requestConsumer call. Temporary change while testing locally
+//        Object mcolSubmitQueryResponse = requestConsumer.submitQuery(submitQueryRequest, connectionTimeOut, requestTimeOut);
+        Object mcolSubmitQueryResponse = null;
+        // TODO: local testing with 1 mcol Result XML default
```

Comment:
remove these comments and uncomment line 438

Reply:
Done